$NetBSD: patch-ap,v 1.1 2007/03/17 13:25:46 rillig Exp $

gcc4 complains:
error: static declaration of 'ctcp_type' follows non-static declaration

--- source/ctcp.c.orig	2003-05-27 09:00:22.000000000 +0200
+++ source/ctcp.c	2007-03-17 14:12:31.000000000 +0100
@@ -176,7 +176,7 @@ CtcpEntryDll *dll_ctcp = NULL;
 
 /* CDE do ops and unban logging */
 
-static char	*ctcp_type[] =
+char	*ctcp_type[] =
 {
 	"PRIVMSG",
 	"NOTICE"
